ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksPage.tsx

```diff
@@ -33,8 +35,13 @@ function ProfileChecksPage() {
                     <ProfileChecksTable
                         isLoading={isLoading}
                         error={error}
-                        profileChecks={profileChecks?.profileResults ?? []}
+                        profileChecksResults={profileChecks?.profileResults ?? []}
+                        profileChecksResultsCount={profileChecks?.totalCount ?? 0}
                         profileName={profileName}
+                        page={page}
+                        perPage={perPage}
+                        setPage={setPage}
+                        setPerPage={setPerPage}
```

Comment:
What do you think about passing the full `pagination` object from `useURLPagination` so we pass less individual props?